test/suites/dev-tanssi-relay/collator-assignment/test-configurable-priority.ts

```diff
@@ -0,0 +1,271 @@
+import "@tanssi/api-augment";
+import { describeSuite, expect, beforeAll } from "@moonwall/cli";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { jumpSessions } from "util/block";
+
+const emptyGenesisData = (api) => {
+    const g = api.createType("DpContainerChainGenesisDataContainerChainGenesisData", {
+        storage: [
+            {
+                key: "0x3a636f6465",
+                value: "0x0102030405060708090a",
+            },
+        ],
+        name: "0x436f6e7461696e657220436861696e2032303030",
+        id: "0x636f6e7461696e65722d636861696e2d32303030",
+        forkId: null,
+        extensions: "0x",
+        properties: {
+            tokenMetadata: {
+                tokenSymbol: "0x61626364",
+                ss58Format: 42,
+                tokenDecimals: 12,
+            },
+            isEthereum: false,
+        },
+    });
+    return g;
+};
+
+const sortCollatorAssignmentByNumberOfCollators = (collatorAssignment) => {
+    return Object.keys(collatorAssignment["containerChains"]).sort((a, b) => {
+        return collatorAssignment["containerChains"][b].length - collatorAssignment["containerChains"][a].length;
+    });
```

Comment:
Maybe mapping the array to Number will allow to use `.to.deep.equal` instead of `toString`?

```suggestion
    }).map((x) => Number(x));
```

Reply:
*fixed*